common/config.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/coreos/rkt/pkg/config"
+)
+
+const (
+	// configuration directory basenames
+	OldCDB = ""
```

Comment:
The `pkg/config` package has a concept of configuration directory basename, which is basically a `CDB` in `/etc/rkt/CDB/someconf.d` path. So, new configuration for auth credentials will be in, say, `/etc/rkt/stage0/auth.d` (CDB here is `stage0`) . But to be backward compatible, we need to take `/etc/rkt/auth.d` into account (CDB here is an empty string).